src/OpenTelemetry.Exporter.Zipkin/Implementation/ZipkinSpan.cs

```diff
@@ -51,8 +51,8 @@ internal class ZipkinSpan
         [JsonProperty("remoteEndpoint")]
         public ZipkinEndpoint RemoteEndpoint { get; set; }
 
-        [JsonProperty("annotations")]
-        public List<ZipkinAnnotation> Annotations { get; set; }
```

Comment:
zipkin is still using name "annotation". This file represents Zipkin's schema, not OpenTelemetry one. Please revert the change in this file.

We really need tests for Zipkin to catch it in unit tests =)